src/cli.ts

```diff
@@ -14,6 +14,11 @@ program
     "JSON string or a Path/Url pointing to an open rpc schema",
     "./openrpc.json",
   )
+  .option(
```

Comment:
What about for rust client? 

IMO this should be a prefix, and the language should be arbitrarily appended to the prefix. For example:

prefix: foobar
generated name foobar-{language}